backend/src/main/java/io/spoud/data/kafka/MatchResultKafkaBO.java

```diff
@@ -21,8 +22,8 @@
   private Integer blueScore;
   private Integer points;
 
-  private UUID blueDefense;
-  private UUID blueOffense;
-  private UUID redDefense;
-  private UUID redOffense;
+  private PlayerEO blueDefense;
+  private PlayerEO blueOffense;
+  private PlayerEO redDefense;
+  private PlayerEO redOffense;
 }
```

Comment:
intended to do this in ksql. would be possible probably but really fucking hard as it stands now

Reply:
yeah, I think also. But we could always materialize the players and a join, but it's easier to do it here I guess